src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -524,6 +524,128 @@ private static IEnumerable<FrameworkRuntimePair> ReadCompatibilitySets(JsonTextR
             });
         }
 
+        internal static void ReadCentralTransitveDependencyGroup(
+            JsonTextReader jsonReader,
+            IList<LibraryDependency> results,
+            string packageSpecPath)
+        {
+            jsonReader.ReadObject(propertyName =>
+            {
+                if (string.IsNullOrEmpty(propertyName))
+                {
+                    // Advance the reader's position to be able to report the line and column for the property value.
+                    jsonReader.ReadNextToken();
+
+                    throw FileFormatException.Create(
+                        "Unable to resolve dependency ''.",
+                        jsonReader.LineNumber,
+                        jsonReader.LinePosition,
+                        packageSpecPath);
+                }
+
+                if (jsonReader.ReadNextToken())
+                {
+                    int dependencyValueLine = jsonReader.LineNumber;
+                    int dependencyValueColumn = jsonReader.LinePosition;
+                    var versionLine = 0;
+                    var versionColumn = 0;
+
+                    var dependencyIncludeFlagsValue = LibraryIncludeFlags.All;
+                    var dependencyExcludeFlagsValue = LibraryIncludeFlags.None;
```

Comment:
Why do we have dependency include flags on PackageVersion?

Isn't that a PackageReference concept?

Reply:
It is a Dependency concept. The CentralTransitveDependencyGroup information is used for Pack scenarios, for this reason the Include/Exclude Assets info is needed.

Reply:
oh I totally spaced here. It's a different type. Gonna start the review again :D